swap_meet/vendor.py

```diff
@@ -1,2 +1,92 @@
 class Vendor:
-    pass
\ No newline at end of file
+    # Wave 1 
+    def __init__(self, inventory=None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        self.item = item
+        self.inventory.append(self.item)
+        return self.item
```

Comment:
This works, but we don't need line 9 here. `self.item = item` would be useful if we needed to re-use that item later, but we only need it for this one function call. So we can just append `item` directly. This is true for all of our methods in this project except for the constructor.